sap_hana/datadog_checks/sap_hana/data/conf.yaml.example

```diff
@@ -84,19 +87,47 @@ instances:
     ##                          use the `count` type to perform aggregation for queries that
     ##                          return multiple rows with the same or no tags.
     ## 3. tags (optional) - A list of tags to apply to each metric.
+    ##
+    ## Notes:
+    ##   - Use the pipe if you require a multi-line script.
+    ##   - Columns without a name are ignored, put `- {}` for any column you wish to skip
     #
     # custom_queries:
-    #   - query: |  # Use the pipe if you require a multi-line script.
-    #       SELECT DATABASE_NAME,
-    #              COUNT(*)
-    #       FROM SYS_DATABASES.M_DATA_VOLUMES
-    #       GROUP BY DATABASE_NAME
+    #   - query: "SELECT DATABASE_NAME,\n        COUNT(*)\nFROM SYS_DATABASES.M_DATA_VOLUMES\n\
```

Comment:
Any suggestion on how to fix it? cc @ofek

Reply:
@coignetp , there is a db template now fyi

Reply:
No way unfortunately